libraries/passive_sensitive.rb

```diff
@@ -7,7 +7,7 @@
 
 class Chef # rubocop:disable Style/MultilineIfModifier
   # Making the sensitive attribute passive for older chef versions
-  class Resource # rubocop:disable Style/Documentation
+  class Resource # rubocop:disable Style/Documentation, Lint/RedundantCopDisableDirective
```

Comment:
Not sure exactly what the deal is here.  Rubocop passed fine locally but kept failing on this Lint check on the Travis builds.  This is the only thing I could make work both locally and on Travis.